### PR TITLE
Release 4.4.3

### PR DESCRIPTION
**Pull request by mfussenegger**
## Summary of the changes / Why this improves CrateDB

🚢

## Checklist

 - [x] Added an entry in `CHANGES.txt` for user facing changes
 - [x] Updated documentation & `sql_features` table for user facing changes
 - [x] Touched code is covered by tests
 - [x] [CLA](https://crate.io/community/contribute/cla/) is signed
 - [x] This does not contain breaking changes, or if it does:
    - It is released within a major release
    - It is recorded in ``CHANGES.txt``
    - It was marked as deprecated in an earlier release if possible
    - You've thought about the consequences and other components are adapted
      (E.g. AdminUI)

### CODE REVIEW

File: docs/appendices/release-notes/4.4.3.rst [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+- Fixed an issue that led to a ``CircuitBreakingException`` when using the
+  ``ANALYZE`` statement.
+
+- Fixed an issue that led to ``Values less than -1 bytes`` errors if ``TRACE``
+  logging was activated for the circuit breaker package.
+
+- Fixed shard allocation on downgraded nodes where only the ``HOTFIX`` version
+  part differs to fully support rolling downgrades to same ``MAJOR.MINOR``
+  versions.
```
Comment (norosa) [inline docs/appendices/release-notes/4.4.3.rst:44]:
```suggestion
- Fixed :ref:`shard allocation <gloss-shard-allocation>` on downgraded nodes
  where only the ``HOTFIX`` version part differs to fully support rolling
  downgrades to same ``MAJOR.MINOR`` versions.
```

**Review by norosa — COMMENTED** [review]
lgtm

File: docs/appendices/release-notes/4.4.3.rst (hunk 1 above)
Comment (mfussenegger) [reply docs/appendices/release-notes/4.4.3.rst:44]: Tried to apply this but can't because the glossary hasn't been ported to 4.4 (which imho is fine)

**Review by seut — APPROVED** [review]
👍

**Pull request merged**